envoy/api/v2/route/route.proto

```diff
@@ -539,6 +539,12 @@ message RedirectAction {
   string host_redirect = 1;
 
   // The path portion of the URL will be swapped with this value.
+  // [#comment:InProgress(bmadhavan)]: Add support for dynamic redirect msg]
```

Comment:
I don't quite follow how this feature works from reading this. Can you clarify and potentially provide a couple of examples in the docs? Also, unless the Envoy PR is ready can we hold this or potentially add support for multi-line comments in the doc parser? (Since I think this will render after the first line.)

Reply:
Sure. I will add more details about the feature & some examples as well. 
Also I am planning on sending the Envoy PR early next week & so we can hold this PR un-till then. Thanks.